pkgs/servers/code-server/default.nix

```diff
@@ -0,0 +1,139 @@
+{ stdenv, fetchFromGitHub, python, zip, makeWrapper, fetchurl, pkgconfig, git
+, rsync, makeDesktopItem, runtimeShell, writeScript, runCommand
+, nodejs, yarn, libsecret, xorg, ripgrep, electron }:
+
+with stdenv.lib;
```

Comment:
Is this even being used outside of the `meta` attribute? It’s a lot more common to use `meta = with stdenv.lib; { … }` than to import it like this at the top level.

If you do need easy access to `lib` it’s also more common to do something like `let inherit (stdenv) lib; in` or just put `lib` in the arguments.

Reply:
Should be fixed now